docs/cubeviz/plugins.rst

```diff
@@ -274,3 +274,22 @@ Export Plot
 ===========
 
 This plugin allows exporting the plot in a given viewer to various image formats.
+
+.. _cubeviz-export-video:
+
+Movie
+-----
+
+.. note::
+
+    For MPEG-4, this feature needs ``opencv-python`` to be installed;
```

Comment:
@camipacifici , is a note here sufficient?

Reply:
So, you want all errors go to this traitlet thingy instead of snackbar?

Reply:
I'd like to see the note in the plugin itself at a minimum, together with the link to the docs.
I did see the red snackbar when I first used it without having installed the additional package.

Reply:
First thing first... do you not see this when opencv-python is not installed and you click "export to MP4"?

![Screenshot 2023-06-29 120827](https://github.com/spacetelescope/jdaviz/assets/2090236/cef6fb75-ee35-440e-a347-5f70fca02231)

Reply:
How about this? Note that I cannot insert HTML links into the text, so I can just do plain text.

![Screenshot 2023-06-29 124030](https://github.com/spacetelescope/jdaviz/assets/2090236/d39f068e-6136-4b0c-a578-f0e800faa7fa)